### ISSUE TITLE
Error when runscript does not exist

### ISSUE THREAD

**Issue by JanStreffing** [esm-tools/esm_tools#659]
When trying to run a runscript that does not exist, one gets this error message. A simple "runscript with the name $NAME does not exist" would be nice.

```
esm_runscripts pi_controll.yaml -e pi_controll_3-1-TCO95_CORE2_open
Traceback (most recent call last):
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_runscripts/config_initialization.py", line 79, in get_user_config_from_command_line
    user_config = esm_parser.initialize_from_yaml(
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_parser/esm_parser.py", line 217, in initialize_from_yaml
    user_config = yaml_file_to_dict(filepath)
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_parser/yaml_to_dict.py", line 217, in yaml_file_to_dict
    raise FileNotFoundError(
FileNotFoundError: All file extensions tried and none worked for /p/project/chhb19/jstreffi/esm_tools/pi_controll.yaml

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/p/home/jusers/streffing1/juwels/.local/bin/esm_runscripts", line 33, in <module>
    sys.exit(load_entry_point('esm-tools', 'console_scripts', 'esm_runscripts')())
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_runscripts/cli.py", line 264, in main
    Setup = SimulationSetup(command_line_config)
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_runscripts/sim_objects.py", line 27, in __init__
    user_config = config_initialization.init_first_user_config(
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_runscripts/config_initialization.py", line 14, in init_first_user_config
    user_config = get_user_config_from_command_line(command_line_config)
  File "/p/project/chhb19/jstreffi/esm_tools/src/esm_runscripts/config_initialization.py", line 89, in get_user_config_from_command_line
    raise("An error occurred reading the config file from the command line") 
TypeError: exceptions must derive from BaseException
```

**Comment by denizural** [comment]
Hi @JanStreffing, 
this is actually in my clean-up TODO list that I mentioned in today's meeting. I previously implemented absolute and relative paths for the runscript but better error handling in case of a missing runscript is also a good idea.

**Comment by nwieters** [comment]
This issue has been raised again and solved in issue #1004.
